pymess/management/commands/send_messages_batch.py

```diff
@@ -69,17 +65,15 @@ def _print_result(self, title, message_pks):
             self.stdout.write('{}: {}'.format(title, len(message_pks)))
 
     def handle(self, type, *args, **options):
-        sender = self.senders[type]()
-
-        if not sender.is_turned_on_batch_sending():
-            raise CommandError('Batch sending is turned off')
+        controller = self.controllers[type]
```

Comment:
the problematic `is_turned_on_batch_sending` part... It's needed to send messages from backend

Reply:
refactored